src/_press/calitp-unveils-new-open-data-standard-to-improve-transit-operations.md

```diff
@@ -0,0 +1,8 @@
+---
+date: "2023-01-04T17:00:00-07:00"
+title: Cal-ITP unveils new Open Data Standard (ODS) to improve transit agency operations
+external: https://www.calitp.org/assets/Cal-ITP.Open.Data.Standard.Improve.Transit.Agency.Operations.pdf
```

Comment:
This is also a duplicate actually 
<img width="1011" alt="image" src="https://github.com/cal-itp/calitp.org/assets/3673236/4b156bfc-e03b-4ae3-bb4c-7498b731eadc">

Removing